includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -368,7 +368,7 @@ public function payment_icons() {
 			WC_Stripe_Payment_Methods::CASHAPP_PAY => '<img src="' . WC_STRIPE_PLUGIN_URL . '/assets/images/cashapp.svg" class="stripe-cashapp-icon stripe-icon" alt="Cash App Pay" />',
 		];
 		$settings  = WC_Stripe_Helper::get_stripe_settings();
-		if ( 'yes' === $settings['single_payment_element'] ) {
```

Comment:
Might it be worth splitting this over multiple lines to keep each line simpler? This takes me more than one pass to fully understand!
```suggestion
		$single_payment_element_setting = $settings['single_payment_element'] ?? null;
		if ( 'yes' === $single_payment_element_setting ) {
```

Reply:
For some reason, I could not automatically apply your suggestion, so I did it manually in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4238/commits/c03800fecb29e3b29538221840e82a904fbd3bcb